### ISSUE TITLE
Failed to achieve the reported verification performance in Table 2 in your paper with your suggestion configuration.

### ISSUE THREAD

**Issue by tianxianhao** [deepinsight/insightface#1867]
I download CASIA-webface dataset from the dataset wiki in this project and convert face images to record file via im2rec.py.
According to your paper, max step is set to 32,000, the start lr is 0.1 and lr decay septs are 20,000 and 28,000. Batch size is 128*4. 
The training environment is V100 *4. network is resnet50. training dataset is casia-webface. val target is lfw and so on.
But the training loss always turned to be nan after few epochs. Is there any suggestion?
 @nttstar

**Comment by tianxianhao** [comment]
face images has been rescale to 112x112. identity number is 10575.
